substrate-node/pallets/pallet-dao/src/dao.rs

```diff
@@ -41,14 +41,18 @@ impl<T: Config> Pallet<T> {
         );
 
         let now = frame_system::Pallet::<T>::block_number();
+        let minimum_threshold = T::MotionMinThreshold::get();
```

Comment:
I would gather everything related to threshold logic at same place just before duration logic:
~~~rust        
        // threshold should be at least the configured minimum threshold for a motion in runtime
        let minimum_threshold = T::MotionMinThreshold::get();
        ensure!(threshold >= minimum_threshold, Error::<T>::ThresholdTooLow);
~~~
or even shorter since it is already explicit:
~~~rust        
        // threshold should be at least the configured minimum threshold for a motion in runtime
        ensure!(threshold >= T::MotionMinThreshold::get(), Error::<T>::ThresholdTooLow);
~~~

And also, any specific reason to create a storage `MotionMinThreshold` item ?
~~~rust        
        // threshold should be at least 5 since created motion should receive a minimum of 5 votes  
        ensure!(threshold >= 5u32, Error::<T>::ThresholdTooLow);
~~~
To make it configurable for tests maybe ?

Reply:
Done!